libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/AztecEditorFragment.java

```diff
@@ -411,7 +411,7 @@ public void appendMediaFile(final MediaFile mediaFile, final String mediaUrl, Im
                 Bitmap bitmapToShow = ImageUtils.getWPImageSpanThumbnailFromFilePath(getActivity(), safeMediaUrl, maxWidth);
                 if (bitmapToShow == null) {
                     ToastUtils.showToast(getActivity(), R.string.error_media_load);
-                    bitmapToShow = BitmapFactory.decodeResource(getResources(), R.drawable.media_image_placeholder);
+                    bitmapToShow = BitmapFactory.decodeResource(getResources(), R.drawable.ic_gridicons_image);
```

Comment:
This doesn't appear to work - I see a different image. Perhaps `decodeResources` doesn't work with vector drawables?

![screenshot_1488332922](https://cloud.githubusercontent.com/assets/3903757/23461442/ef87f87e-fe57-11e6-9d0e-779e751e2729.png)